### PR TITLE
Feature/us5287 assign child to room

### PR DESCRIPTION
**Pull request by timmyg**

### CODE REVIEW

File: SignInCheckIn/MinistryPlatform.Translation/Models/DTO/MpEventParticipantDto.cs [hunk 1]
```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using MinistryPlatform.Translation.Models.Attributes;
+using Newtonsoft.Json;
+
+namespace MinistryPlatform.Translation.Models.DTO
+{
+
+                //"Event_Participants_ID_Table.Registrant_Message_Sent"
```
Comment (IngageKrizJ) [inline SignInCheckIn/MinistryPlatform.Translation/Models/DTO/MpEventParticipantDto.cs]: Commented code

File: SignInCheckIn/MinistryPlatform.Translation/Models/DTO/MpKioskConfigDto.cs [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+namespace MinistryPlatform.Translation.Models.DTO
+{
+    [MpRestApiTable(Name = "cr_Kiosk_Configs")]
+    public class MpKioskConfigDto
+    {
+        [JsonProperty(PropertyName = "Kiosk_Config_ID")]
+        public int KioskConfigId { get; set; }
+
+        // TODO: Determine if this actually works - casting to a guid
```
Comment (IngageKrizJ) [inline SignInCheckIn/MinistryPlatform.Translation/Models/DTO/MpKioskConfigDto.cs]: I assume this worked - if so, remove TODO

File: SignInCheckIn/MinistryPlatform.Translation/Repositories/ChildSigninRepository.cs.orig [hunk 3]
```diff
@@ -0,0 +1,186 @@
+﻿using System.Collections.Generic;
+using MinistryPlatform.Translation.Models.DTO;
+using MinistryPlatform.Translation.Repositories.Interfaces;
+using System.Linq;
+using Crossroads.Utilities.Services.Interfaces;
+
+namespace MinistryPlatform.Translation.Repositories
+{
+    public class ChildSigninRepository : IChildSigninRepository
```
Comment (IngageKrizJ) [inline SignInCheckIn/MinistryPlatform.Translation/Repositories/ChildSigninRepository.cs.orig]: Remove this .orig file?

File: SignInCheckIn/MinistryPlatform.Translation/Repositories/ChildSigninRepository.cs [hunk 4]
```diff
@@ -140,5 +140,34 @@ public int GetHashCode(MpParticipantDto obj)
                 return $"{obj.ParticipantId}{obj.ContactId}".GetHashCode();
             }
         }
+
+	    public void CreateEventParticipants(List<MpEventParticipantDto> mpEventParticipantDtos)
```
Comment (dtkocher) [inline SignInCheckIn/MinistryPlatform.Translation/Repositories/ChildSigninRepository.cs]: fix formating

File: SignInCheckIn/MinistryPlatform.Translation/Repositories/ChildSigninRepository.cs.orig [hunk 5]
```diff
@@ -0,0 +1,186 @@
...
+                "Group_ID_Table_Ministry_ID_Table.Ministry_ID"
+            };
+
+            var participantIds = string.Join(",", children.Select(x => x.ParticipantId));
+
+            return _ministryPlatformRestRepository.UsingAuthenticationToken(apiUserToken).
+                        SearchTable<MpParticipantDto>("Group_Participants", $"Participant_ID_Table.[Participant_ID] IN ({participantIds}) AND Group_ID_Table_Congregation_ID_Table.[Congregation_ID] = {_applicationConfiguration.KidsClubCongregationId} AND Group_ID_Table_Group_Type_ID_Table.[Group_Type_ID] = {_applicationConfiguration.KidsClubGroupTypeId} AND Group_ID_Table_Ministry_ID_Table.[Ministry_ID] = {_applicationConfiguration.KidsClubMinistryId}", columnList);
+        }
+
+<<<<<<< HEAD
```
Comment (dtkocher) [inline SignInCheckIn/MinistryPlatform.Translation/Repositories/ChildSigninRepository.cs.orig]: there is a merge conflict here that wasn't properly resloved.

File: SignInCheckIn/SignInCheckIn/Controllers/ChildSigninController.cs [hunk 6]
```diff
@@ -33,5 +44,22 @@ public IHttpActionResult GetEvents(string phoneNumber)
                 throw new HttpResponseException(apiError.HttpResponseMessage);
             }
         }
+
+        [HttpPost]
+        //[ResponseType(typeof(List<ParticipantDto>))]
```
Comment (IngageKrizJ) [inline SignInCheckIn/SignInCheckIn/Controllers/ChildSigninController.cs]: Commented attribute

File: SignInCheckIn/SignInCheckIn/Models/DTO/KioskConfigDto.cs [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+using System.Web;
+using Newtonsoft.Json;
+
+namespace SignInCheckIn.Models.DTO
+{
+    public class KioskConfigDto
+    {
+        public int KioskConfigId { get; set; }
+
+        // TODO: Determine if this actually works - casting to a guid
```
Comment (IngageKrizJ) [inline SignInCheckIn/SignInCheckIn/Models/DTO/KioskConfigDto.cs]: I assume this is working, if so, remove TODO

**Pull request merged**